l10n_ch_payment_return_sepa/demo/demo.yml

```diff
@@ -0,0 +1,505 @@
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+# Create in invoices
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+-
+  !record {model: account.invoice, id: ct_invoice_1, view: False}:
```

Comment:
Yaml is no more supported. Remove the file and make sure nothing is missing for the tests. If needed, convert this to xml